### ISSUE TITLE
JavaFX noisy warnings in the console (java.lang.ClassCastException)

### ISSUE THREAD

**Issue by qertoip** [sparrowwallet/sparrow#76]
On the console there is a huge list of exceptions similar to:

`WARNING: Caught 'java.lang.ClassCastException: class java.lang.String cannot be cast to class javafx.scene.paint.Color (java.lang.String is in module java.base of loader 'bootstrap'; javafx.scene.paint.Color is in module javafx.graphics of loader 'app')' while converting value for '-fx-background-color' from rule '*.toggle-switch:selected *.thumb-area' in stylesheet jrt:/org.controlsfx.controls/org/controlsfx/control/toggleswitch.css`

It does not seem to hinder anything - the wallet does work despite these warnings.

**Comment by craigraw** [comment]
Indeed, they are noisy but harmless. I've tried to quieten them with 34080ae, although I suspect they are being written directly to the console may bypass the logging framework.

**Comment by craigraw** [comment]
Should be properly sorted now - I had to redirect JUL logging to the SLF4J framework, and set the log level there to suppress them.

**Comment by craigraw** [comment]
I think this is sorted now - closing this off.
